substrate/frame/offences/benchmarking/src/inner.rs

```diff
@@ -221,9 +227,9 @@ mod benchmarks {
 	) -> Result<(), BenchmarkError> {
 		// for grandpa equivocation reports the number of reporters
 		// and offenders is always 1
-		let reporters = vec![account("reporter", 1, SEED)];
+		let reporter = account::<T::AccountId>("reporter", 1, SEED);
 
-		// make sure reporters actually get rewarded
+		// make sure reporter actually get rewarded
```

Comment:
```suggestion
		// make sure reporter actually gets rewarded
```

Don't think it's actually from this PR, but can't resist